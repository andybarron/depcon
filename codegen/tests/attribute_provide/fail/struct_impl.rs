use depcon::*;

struct Bad;

#[provide]
impl Bad {}

fn main() {}
