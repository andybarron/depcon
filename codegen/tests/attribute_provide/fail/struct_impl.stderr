error: custom attribute panicked
 --> tests/attribute_provide/fail/struct_impl.rs:5:1
  |
5 | #[provide]
  | ^^^^^^^^^^
  |
  = help: message: attribute must be applied to a trait impl
